cpp/src/parquet/metadata.h

```diff
@@ -523,10 +523,14 @@ struct PageIndexLocation {
 
 class PARQUET_EXPORT FileMetaDataBuilder {
  public:
-  // API convenience to get a MetaData reader
+  ARROW_DEPRECATED("Deprecated in 12.0.0. Use overload without KeyValueMetadata instead.")
```

Comment:
```suggestion
  ARROW_DEPRECATED("Deprecated in 13.0.0. Use overload without KeyValueMetadata instead.")
```
Should I change this since it may not be included in the 12.0.0 release? @wjones127